release-notes/opensearch-security-dashboards-plugin.release-notes-1.3.16.0.md

```diff
@@ -0,0 +1,7 @@
+## 2023-12-08 Version 1.3.14.0
+
+Compatible with OpenSearch-Dashboards 1.3.14
+
+### Enhancement
```

Comment:
This can be classified as a bug fix. Please update the versions in this file to 1.3.16.0 and remove the dates.

Reply:
fixed.
cc: @DarshitChanpura